jinahub/indexers/dbms/LMDBDBMSIndexer/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM jinaai/jina:1.2.0
```

Comment:
1.2?

Reply:
doesn't matter. It's installed from `requirements.txt`